[2023-03-22T04:14:37.586+0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: test_15.format_to_parquet_task scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-22T04:14:37.589+0400] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: test_15.format_to_parquet_task scheduled__2023-03-21T00:00:00+00:00 [queued]>
[2023-03-22T04:14:37.589+0400] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T04:14:37.589+0400] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2023-03-22T04:14:37.589+0400] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T04:14:37.608+0400] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): format_to_parquet_task> on 2023-03-21 00:00:00+00:00
[2023-03-22T04:14:37.612+0400] {standard_task_runner.py:55} INFO - Started process 184283 to run task
[2023-03-22T04:14:37.614+0400] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'test_15', 'format_to_parquet_task', 'scheduled__2023-03-21T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmp8ojf0m6w']
[2023-03-22T04:14:37.615+0400] {standard_task_runner.py:83} INFO - Job 6: Subtask format_to_parquet_task
[2023-03-22T04:14:37.649+0400] {task_command.py:388} INFO - Running <TaskInstance: test_15.format_to_parquet_task scheduled__2023-03-21T00:00:00+00:00 [running]> on host civispc
[2023-03-22T04:14:37.674+0400] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=test_15
AIRFLOW_CTX_TASK_ID=format_to_parquet_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-21T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-21T00:00:00+00:00
[2023-03-22T04:14:37.674+0400] {logging_mixin.py:137} INFO - ---------------------ENV-------------------------
[2023-03-22T04:14:37.674+0400] {logging_mixin.py:137} INFO - no cred((((((((((((((((((
[2023-03-22T04:14:37.674+0400] {logging_mixin.py:137} INFO - /home/civis/git/de_zoomcamp_project/airflow
[2023-03-22T04:14:37.674+0400] {logging_mixin.py:137} INFO - /home/civis/git/de_zoomcamp_project/airflow
[2023-03-22T04:14:37.674+0400] {logging_mixin.py:137} INFO - Files and directories:
[2023-03-22T04:14:37.674+0400] {logging_mixin.py:137} INFO - ['cred.json', 'webserver_config.py', 'dags', 'logs', 'airflow.cfg', 'airflow.db', 'airflow-webserver.pid', 'pldb.csv']
[2023-03-22T04:14:37.676+0400] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/civis/anaconda3/lib/python3.9/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/civis/anaconda3/lib/python3.9/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/civis/git/de_zoomcamp_project/airflow/dags/test.py", line 44, in format_to_parquet
    cred = Variable.get("cred")
  File "/home/civis/anaconda3/lib/python3.9/site-packages/airflow/models/variable.py", line 141, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable cred does not exist'
[2023-03-22T04:14:37.677+0400] {taskinstance.py:1318} INFO - Marking task as FAILED. dag_id=test_15, task_id=format_to_parquet_task, execution_date=20230321T000000, start_date=20230322T001437, end_date=20230322T001437
[2023-03-22T04:14:37.688+0400] {standard_task_runner.py:100} ERROR - Failed to execute job 6 for task format_to_parquet_task ('Variable cred does not exist'; 184283)
[2023-03-22T04:14:37.705+0400] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-03-22T04:14:37.721+0400] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
